### PR TITLE
Wizard pauses

### PR DESCRIPTION
**Pull request by dwhswenson**
This PR will address the need for pauses in the wizard. Currently, the wizard hops from one step to the next very quickly; a brief pause (and possibly some visual cues) will improve UX by making it easier to track what is happening.

This addresses one of the targets for before v0.3 raised in #57:

* Add pauses at various points to give better UX
  * creating subvolumes
  * after naming an object

Tasks
- [x] Write basic pausing module, which can be reused (and which allows customization of delays)
- [x] Tests for pausing module
- [x] Add pause before creating subvolumes
- [x] Add pause after each object is created
- [x] Fix unit tests so that `'nopause'` pause style is used in test suite

### CODE REVIEW

**Comment by dwhswenson** [comment]
This is ready for review and comment. I will leave it open for at least 24 hours, merging no earlier than Tue 02 Nov 18:00 GMT (14:00 my local).

I think this is an improvement over the previous UX, but I'm very much open to other UX improvements along these lines.

File: paths_cli/wizard/pause.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    try:
+        set_pause_style(style)
+        yield
+    finally:
+        set_pause_style(old_style)
+
+
+def get_pause_style():
+    """Get the current pause style"""
+    global _PAUSE_STYLE
```
Comment (sroet) [inline paths_cli/wizard/pause.py:35]: is `global` required here? 
```suggestion
```
(pylint claims:  `Using global for '_PAUSE_STYLE' but no assignment is done (global-variable-not-assigned)`)

**Review by sroet — APPROVED** [review]
1 question about the use of global, otherwise LGTM

File: paths_cli/wizard/pause.py (hunk 1 above)
Comment (dwhswenson) [reply paths_cli/wizard/pause.py:35]: Yeah, that seems to work. I was surprised to need `global`, but I guess I only needed it for setting.

Comment (sroet) [reply paths_cli/wizard/pause.py]: Yeah, for setting it defaults to inner scope, but for returning it first searches the inner scope but then goes to module scope

**Pull request merged**